CMakeLists.txt

```diff
@@ -307,3 +307,29 @@ else()
     target_compile_options(gtda_cech_complex PUBLIC $<$<CONFIG:DEBUG>:-O2 -ggdb -D_GLIBCXX_DEBUG>)
 endif()
 
+target_compile_definitions(gtda_cech_complex PRIVATE BOOST_RESULT_OF_USE_DECLTYPE=1 BOOST_ALL_NO_LIB=1 BOOST_SYSTEM_NO_DEPRECATED=1)
```

Comment:
Why it was necessary to add this `target_compile_definitions`, it's already in original file

Reply:
I will fix it, I believe it is  mistake that happened during merging!